src/shared/cards/Card Template.js

```diff
@@ -0,0 +1,73 @@
+import Card from 'cards/Card';
+import { Set } from 'immutable';
+
+export default class CardName extends Card {
+  name = 'Card Name';
```

Comment:
because of a weird thing in JS this can't be called `name` so I changed it to `title`

Reply:
also all of these should be `static`
```
static title = 'Card Name';